modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,37 @@
+[id="enabling-ang-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+
+.Procedure
+. To give {product-short} access to GitHub, link:{linkgettingstartedguide}#configuring-github-integration[configure GitHub integration].
+
+. The Bulk Import plugins are installed but disabled by default.
+To enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` plugins,
+edit your `dynamic-plugins.yaml` with following content.
+See link:{plugins-configure-book-url}[{plugins-configure-book-title}].
++
+.`dynamic-plugins.yaml` fragment
+[source,yaml]
+----
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
+        disabled: false
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import
+        disabled: false
+----
+
+. Only {product-short} administrators or users with the `bulk.import` permission can use the Bulk Import feature.
+Configure the required `bulk.import` RBAC permission for the users that are not administrators.
+See link:{authorization-book-url}#ref-rbac-permission-policies_title-authorization[Permission policies in Red Hat Developer Hub].
```

Comment:
I realize that https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/authorization/index#ref-rbac-permission-policies_title-authorization does not list "Bulk Import permissions" as policies supported in Developer Hub. 
I think it should be added to the list as well, similar to the tables for Scaffolder, Topology, OCM, ... permissions.
Here is the upstream table: https://github.com/janus-idp/backstage-plugins/blob/release-1.3/plugins/rbac-backend/docs/permissions.md#bulk-import

Reply:
Added